test/code/plugins/fake_npmd_binary.rb

```diff
@@ -6,6 +6,7 @@ class NPMDTest
     FAKE_PATH_DATA  = '{"DataItems":[{"SourceNetwork":"abcd", "SourceNetworkNodeInterface":"abcd", "SourceSubNetwork":"abcd", "DestinationNetwork":"abcd", "DestinationNetworkNodeInterface":"abcd", "DestinationSubNetwork":"abcd", "RuleName":"abcd", "TimeSinceActive":"abcd", "LossThreshold":"abcd", "LatencyThreshold":"abcd", "LossThresholdMode":"abcd", "LatencyThresholdMode":"abcd", "SubType":"NetworkPath", "HighLatency":"abcd", "MedianLatency":"abcd", "LowLatency":"abcd", "LatencyHealthState":"abcd","Loss":"abcd", "LossHealthState":"abcd", "Path":"abcd", "Computer":"abcd"}]}'
 
     FAKE_AGENT_DATA = '{"DataItems":[{"AgentFqdn":"abcd", "AgentIP":"abcd", "AgentCapability":"abcd", "SubnetId":"abcd", "PrefixLength":"abcd", "AddressType":"abcd", "SubType":"NetworkAgent", "AgentId":"abcd"}]}'
+    FAKE_AGENT_DATA2 = '{"DataItems":[{"AgentFqdn":"abcd", "AgentIP":"abcd", "AgentCapability":"abcd", "SubnetId":"abcd", "PrefixLength":"abcd", "AddressType":"abcd", "SubType":"NetworkAgent", "AgentId":"abcd"}]}'
```

Comment:
Is there any difference between FAKE_AGENT_DATA and FAKE_AGENT_DATA2? if not in the statement on line 40 we can again call puts with FAKE_AGENT_DATA